content/en/case-studies/blackhole-image.md

```diff
@@ -0,0 +1,79 @@
+
+### Key Imaging Objectives
+
+* To study the most extreme objects in the Universe predicted by Einstein’s theory of general relativity, during the centennial year of the historic experiment that first confirmed the theory (2017).
+
+* Focus on the [black hole](https://solarsystem.nasa.gov/resources/2319/first-image-of-a-black-hole/) at the center of Messier 87 galaxy, located in the Virgo galaxy cluster. This black hole resides approximately 55 million light-years from Earth and has a mass equal  to 6.5 billion times that of the Sun. It has been a subject of astronomical study for [over a 100 years](https://www.jpl.nasa.gov/news/news.php?feature=7385). Black holes have been theoretically predicted and observed but a real image was never created until now.
+
+* Based on Einstein’s general theory of relativity, the scientists expected to see a dark region similar to a shadow, caused by the gravitational bending and capture of light by the event horizon. By studying this shadow scientists could measure the enormous mass of M87’s black hole. 
+
+### The Challenges 
+
+* **Scale**
+
+    The observations from Event Horizon Telescope (EHT) present challenges for existing data processing tools, arising from the rapid atmospheric phase fluctuations, wide recording bandwidth, and highly heterogeneous array.
+
+* **Calibration and Correlation**
+
+    Besides scheduling all of these coordinated observations of EHT, reducing the overall volume and complexity of data to aid analysis is a really hard problem to solve. To put things in perspective, EHT generates over 350 Terabytes worth of observed data per day, stored on high-performance helium filled hard drives.
+
+* **Source Imaging and Model Fitting**
+
+    The sequence of correlation and engineering releases represents a year-long effort of identifying and mitigating data issues, and developing new software and procedures that could reliably choose the most likely image based on actual measurements.
+
+<img
+  src="/images/content_images/cs/dataprocessbh.png"
+  alt="dpipe"
+  title="Data Processing Pipeline">
+
+Figure: Data processing pathway of an EHT observation from recording to image (or other physical parameters) estimation (Source: [IOPScience Paper](https://iopscience.iop.org/article/10.3847/2041-8213/ab0c57))
+
+## NumPy’s Role in Black Hole Imaging
+	
+There are several aspects to black hole imaging besides data collection, noise elimination, data cleanup, reduction and correlation. Imaging is crucial as it can help  to predict not only the black hole mass but also rule out whether a black hole could be a wormhole, a theoretical bridge between distant points in spacetime. But it is also incredibly hard to measure given the astronomical distances involved. As Katie Bouman mentions in her [TED talk](https://www.youtube.com/watch?v=BIvezCVcsYs), ‘It is like taking a picture of an orange on the surface of the moon.’
+
+<img
+  src="/images/content_images/cs/bh_numpy_role.png"
```

Comment:
The `NumPy showcase` title probably should be cropped from this image.

Reply:
done.